FHEM/00_SIGNALduino.pm

```diff
@@ -1201,28 +1200,35 @@ sub SIGNALduino_CheckCcregResponse {
   my $hash = shift;
   my $msg = shift;
   my $name=$hash->{NAME};
-
+  $hash->{logMethod}->($name, 5, "$name: CheckCcregResponse, msg $msg");
   if ($msg =~ /^ccreg/) {
-
-    $msg =~ s/\s\sccreg/\nccreg/g;
-    $msg =~ s/ccreg\s\d0:\s//g;
-
-    my @ccreg = split(/\s/,$msg);
-
-    $msg.= "\n\n";
-    $msg.= "Configuration Register Detail (address, name, value):\n";
-
+    my $msg1 = $msg;
+    $msg =~ s/\s\s/\n/g;
+    $msg = "\nConfiguration register overview:\n---------------------------------------------------------\n" . $msg;
+    $msg.= "\n\nConfiguration register detail:\n---------------------------------------------------------\nadd.  name       def.   cur.\n";
+    $msg1 =~ s/ccreg\s\d0:\s//g;
```

Comment:
Irgendwas stimmt hier nicht mit dem entfernen vom Text ccreg.
Das merkt der Test und schlägt daher auch fehl, weil ccreg 00: weiterhin in der Rückgabe enthalten ist.

Reply:
Ich habe "ccreg 00:" wieder mit rein genommen, weil ich die Ausgabe in dieser Form https://github.com/RFD-FHEM/RFFHEM/pull/918#issuecomment-750466589 übersichtlicher finde.

Reply:
![image](https://user-images.githubusercontent.com/7968127/103171819-70fa4c00-484f-11eb-97a8-dbafd5e21d9e.png)

Es geht um das rot markierte aus meiner Sicht. Bezog sích deine Aussage darauf auch?
Ich denke wir haben das ccreg ## entfernt, weil
a) ccreg schon weiter oben steht und die ## nicht näher definiert sind.

PS: Die Regex sucht jede Zeile einzeln, daher brauchst Du keine Zeilenumbrüche in die Regex selbst einbauen.

Reply:
Ich bezog mich auch darauf.  Ich finde es so übersichtlicher und wenn und da es einmal so ankommt, finde ich es auch unnötig, das wieder zu entfernen.
Das mit den Zeilenumbrüchen habe ich auch bemerkt und eben korrigiert.

Jetzt hängt der Test woanders:
```
not ok 5 - Test SIGNALduino_CheckVersionResp with no version {
    1..1
    not ok 1 - check return message
}
not ok 6 - Test SIGNALduino_CheckVersionResp timeout {
    1..1
    not ok 1 - check return message
}

    # Failed test 'check return message'
Error:     # at (eval 102)[./FHEM/98_unittest.pm:196] line 33.
    # +-----+----+---------------------------------------------------------+
    # | GOT | OP | CHECK                                                   |
    # +-----+----+---------------------------------------------------------+
    # | 2   | =~ | (?^u:^dummyDuino: CheckVersionResp, Not an SIGNALduino  |
    # |     |    | device, got for V: no version from device)              |
    # +-----+----+---------------------------------------------------------+
# Failed test 'Test SIGNALduino_CheckVersionResp with no version'
Error: # at (eval 102)[./FHEM/98_unittest.pm:196] line 34.
    # Failed test 'check return message'
Error:     # at (eval 102)[./FHEM/98_unittest.pm:196] line 43.
    # +-----+----+---------------------------------------------------------+
    # | GOT | OP | CHECK                                                   |
    # +-----+----+---------------------------------------------------------+
    # | 2   | =~ | (?^u:^dummyDuino: CheckVersionResp, Not an SIGNALduino  |
    # |     |    | device, got for V: undef$)                              |
    # +-----+----+---------------------------------------------------------+
# Failed test 'Test SIGNALduino_CheckVersionResp timeout'
Error: # at (eval 102)[./FHEM/98_unittest.pm:196] line 44.
```

Reply:
Seltsam.

Hier ging es noch: (fc4b9c781cfd8d7d84f8e4397e5e6b07e14143fa)
https://github.com/RFD-FHEM/RFFHEM/runs/1573343360?check_suite_focus=true#step:12:9134

Ab hier lief es dann nicht mehr: (714e519ae3baf2fc29eca0ecf3285e3981c55c08)
https://github.com/RFD-FHEM/RFFHEM/runs/1585754386?check_suite_focus=true#step:12:298

In dem commit konnte ich jetzt keine Änderung finden die damit zusammenhängt.

Reply:
Ich denke der Fehler kommt durch eine FHEM Änderung zustande. Ich weiss noch nicht welche, aber ich suche mal